src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx

```diff
@@ -23,12 +21,9 @@ const ArticlesRail: React.FC<ArticlesRailProps> = ({
       <Flex justifyContent="space-between" alignItems="center">
         <Text variant="title">Articles</Text>
         <RouterLink to={`/partner2/${partnerSlug}/articles`}>
-          <Media greaterThan="xs">
-            <ViewAllButton offset={NAV_BAR_HEIGHT} />
-          </Media>
-          <Media at="xs">
-            <ViewAllButton offset={MOBILE_NAV_HEIGHT} />
-          </Media>
+          <ScrollToPartnerHeader>
```

Comment:
This can use `Clickable` + the scroll function itself. Since `Clickable` is a button we don't need to add the `onKeyPress` handler.